.github/workflows/validate-upstream.yml

```diff
@@ -73,7 +76,31 @@ jobs:
         if: ${{ inputs.data-files-id != '' && inputs.data-files-folder != '' }}
         with:
           name: ${{ inputs.data-files-id }}
-          path: ./_data/${{ inputs.data-files-folder }}
+          path: /tmp/_data/${{ inputs.data-files-folder }}
+      -
+        name: Copy data files
+        if: ${{ inputs.data-files-id != '' && inputs.data-files-folder != '' }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+            const dataFilesPlaceholderFolder = `${{ inputs.data-files-placeholder-folder }}`;
+            const globber = await glob.create(`/tmp/_data/${{ inputs.data-files-folder }}/*.yaml`);
+            for await (const yamlSrcPath of globber.globGenerator()) {
+              const yamlSrcFilename = path.basename(yamlSrcPath);
+              const yamlDestPath = path.join('_data', `${{ inputs.data-files-folder }}`, yamlSrcFilename);
+              const placeholderPath = path.join(dataFilesPlaceholderFolder, yamlSrcFilename.replace(/^docker_/, '').replace(/\.yaml$/, '.md'));
+              if (dataFilesPlaceholderFolder !== '' && !fs.existsSync(placeholderPath)) {
+                const placeholderContent = `---\ndatafolder: ${{ inputs.data-files-folder }}\ndatafile: ${yamlSrcFilename.replace(/\.[^/.]+$/, '')}\ntitle: ${yamlSrcFilename.replace(/\.[^/.]+$/, "").replaceAll('_', ' ')}\n---\n{% include cli.md datafolder=page.datafolder datafile=page.datafile %}`;
```

Comment:
Could we split this up into multiple lines? Creating an array first, then joining with newlines?

It's a bit tricky to read at the moment.

Reply:
> Could we split this up into multiple lines?

:+1:

___

Didn't want to waste time with templating or handlebars for this stuff but we can look at this in follow-up to clean this up a bit.